### DISCUSSION TITLE
System Design: The complete course

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#33] [Comments]
# System Design: The complete course

Learn how to design systems at scale and prepare for system design interviews

https://www.karanpratapsingh.com/blog/system-design-the-complete-course#public

**Comment by debakarr** [comment]
Not sure why but the date for this blog shows Sunday, September 4, 2022

**Reply by karanpratapsingh** [reply]
Hi @Dibakarroy1997, that was the planned release date of this article.
